composer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "wordpress/twentynineteen",
+  "description": "A new Gutenberg-ready theme.",
+  "type": "package",
+  "license": "GPL v2 or later",
+  "keywords": [
+		"WordPress",
+		"Themes"
+	],
+  "authors": [
+		{
+			"name": "Contributors",
+			"homepage": "https://github.com/WordPress/twentynineteen/graphs/contributors"
```

Comment:
Also, per `style.css`, the home page should be `https://github.com/WordPress/twentynineteen`

https://github.com/WordPress/twentynineteen/blob/master/style.css#L4